analyses/survival-analysis/survival-analysis_tp53_telomerase.Rmd

```diff
@@ -92,7 +92,15 @@ histologies_rna <- read_tsv(metadata_file, guess_max = 10000) %>%
     sample_id == "7316-116" ~ "Biopsy only",
     sample_id == "7316-182" ~ "Partial resection",
     TRUE ~ as.character(extent_of_tumor_resection))) %>%
```

Comment:
7316-356 will need a recode here as well @sjspielman

Reply:
Ugh, I did that too but didn't commit it 🤦‍♀️

Reply:
Nope, I did commit it! I just put it down here: https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/ac03b3f39e1958cbd503e3ff9cd84e8f6c0423f0/analyses/survival-analysis/survival-analysis_tp53_telomerase.Rmd#L130-L133

I should put it with the rest though, so will update.

Reply:
Oh, I see. Thanks, yeah we should keep sample-specific.